src/app/websites/website-services/afterdark/afterdark.service.ts

```diff
@@ -0,0 +1,201 @@
+import { Injectable } from '@angular/core';
+import { Website } from '../../decorators/website-decorator';
+import { BaseWebsiteService } from '../base-website-service';
+import { WebsiteService, WebsiteStatus, LoginStatus, SubmissionPostData, PostResult } from '../../interfaces/website-service.interface';
+import { Submission, SubmissionFormData } from 'src/app/database/models/submission.model';
+import { AfterDarkSubmissionForm } from './components/afterdark-submission-form/afterdark-submission-form.component';
+import { SubmissionType, SubmissionRating } from 'src/app/database/tables/submission.table';
+import { LoginProfileManagerService } from 'src/app/login/services/login-profile-manager.service';
+import { fileAsFormDataObject } from 'src/app/utils/helpers/file.helper';
+import { TypeOfSubmission } from 'src/app/utils/enums/type-of-submission.enum';
+import { Folder } from '../../interfaces/folder.interface';
+
+const ACCEPTED_FILES = ['png', 'jpeg', 'jpg'];
+
+function submissionValidate(submission: Submission, formData: SubmissionFormData): any[] {
+  var tags = [];
+  if (formData.defaults && formData.defaults.tags) {
+    tags = formData.defaults.tags.tags;
+  }
+  if (tags.length < 2) {
+    return [['AfterDark requires at least two tags', { website: 'AfterDark', value: tags.length }]];
```

Comment:
Incorrect format.
Check weasyl.service.ts to see how tag string should be (because of translation).

Reply:
Switched to correct string